pkg/dataplane/test/sync_test.go

```diff
@@ -1137,6 +1139,49 @@ func (suite *syncStreamTestSuite) TestStream() {
 
 	response.Release()
 
+	getItemsInput := v3io.GetItemsInput{
+		Path:               streamPath,
+		ReturnData:         "true",
+		AllowObjectScatter: "true",
+		AttributeNames:     []string{"**"},
+	}
+
+	suite.populateDataPlaneInput(&getItemsInput.DataPlaneInput)
+
+	cursor, err := v3io.NewItemsCursor(suite.container, &getItemsInput)
+	suite.Require().NoError(err, "Failed to get items")
+
+	cursorItems, err := cursor.AllSync()
+	suite.Require().NoError(err)
+
+	// internal use case test
```

Comment:
Since this is an open source repository, I wouldn't declare this as `internal`
What's preventing regular user from doing the same? Root permission?

Reply:
doesn't have any preventing method,
as for root permissions, Gibby could run with non root user